jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -40,6 +44,8 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
         data_label = [data_label]
     elif isinstance(data, SpectrumList):
         pass
```

Comment:
The changelog says Specviz can now accept `SpectrumList` objects but there is a `pass` here?  So does it support `SpectrumList` or just a list of files, which is what line 47-49 looks like.

Reply:
Great question, I'll see if it works to just drop in a list to fill in this case.

Reply:
Loading a `SpectrumList` does work, I assume the `pass` here is just a reminder that `SpectrumList` is an acceptable input, but one that we don't have to do any special processing for.

Reply:
This seems to be my fault. The changelog entry should read instead "Specviz can load a list of spectra and combine...". The requirement was (at that time) to read that, and not a `SpectrumList` instance. I think because `SpectrumList` was still in some sort of unfinished state.

Reply:
If the regular parser can already handle `SpectrumList` then in that case, it might be good to add a code comment at line 42 mentioning why there's a `pass` there, for posterity.  The scope of the PR here is fine.  I think there should be some more documentation to make it clear for the user what are acceptable inputs, formats, usage, etc.